documentation/guides/migrating-from-v5-to-v6.md

```diff
@@ -0,0 +1,245 @@
+        people. Let shoppers buy from your store without leaving
+        Instagram.
+      </p>
+    </TextContainer>
+  </Modal.Section>
+</Modal>
+```
+
+## Sheet requires an accessibility label
+
+Accessibility labels are required for screen readers. This new label does not render anything to the page.
+
+```diff
+<Sheet
+  open={sheetActive}
+  onClose={toggleSheetActive}
++ accessibilityLabel="Reach more shoppers with Instagram product tags"
+>
+  <TextContainer>
+    <p>
+      Use Instagram posts to share your products with millions of
+      people. Let shoppers buy from your store without leaving
+      Instagram.
+    </p>
+  </TextContainer>
+</Sheet>
+```
+
+## Removed exported type`AnimationProps`
+
+`AnimationProps` is a simple type that was used once in `@shopify/polaris` and we have decided to remove it.
+
+```diff
+- import {AnimationProps} from '@shopify/polaris';
+
++ interface AnimationProps {
++   in?: boolean;
++ }
+```
+
+## Removed export for `NewDesignLanguageColor` and `Color`
+
+We no longer support `NewDesignLanguageColor` and `Color` types. They were used to type our icon colors and have been replaced with ` IconProps['color']`.
+
+```diff
+- import {Color, NewDesignLanguageColor, Icon} from '@shopify/polaris';
++ import {Icon, IconProps} from '@shopify/polaris';
+
+interface PartyIconProps {
+- color: Color | NewDesignLanguageColor;
++ color: IconProps['color'];
+}
+```
+
+## Replaced `BaseAction` type with `Action`
+
+`BaseAction` has been replaced with the `Action` type.
+
+```diff
+- import {BaseAction} from '@shopify/polaris';
++ import {Action} from '@shopify/polaris';
+
+export interface Props {
+- headerAction?: BaseAction;
++ headerAction?: Action;
+}
+```
+
+## Changed `frameOffset` prop to accept a string
+
+This allows relative units for the frame offset. Allowing users to take into account for font-size and matching up breakpoints.
+
+```diff
+<AppProvider
+  i18n={{}}
+  theme={{
+-   frameOffset: 60,
++   frameOffset: '60px',
+  }}
+>
+```
+
+## Replace `ariaPressed` prop with `pressed` in `Button` and `UnstyledButton`
+
+```diff
+- <Button ariaPressed={true}>Let’s go</Button>
++ <Button pressed={true}>Let’s go</Button>
+```
+
+## Removed `Button`'s `stretchContent` prop
+
+Consumers should combine the `fullWidth` and `textAlign="left"` props instead.
+
+```diff
+- <Button stretchContent>Lets go</Button>
++ <Button fullWidth textAlign="left">Lets go</Button>
+```
+
+## Removed `Popover`/`PopoverOverlay`'s `preventAutoFocus` prop
+
+There was duplicated functionality and we have decided to replace `preventAutofocus` with `autofocusTarget="none"`.
+
+```diff
+<Popover
+  active={popoverActive}
+  activator={activator}
+  onClose={togglePopoverActive}
+- preventAutofocus
++ autofocusTarget="none"
+>
+  <p>Hello world</p>
+</Popover>
+```
+
+## Removed `button-filled-disabled` SASS mixin
+
+This mixin has been removed as `Button` `disabled` property implements this background.
```

Comment:
It seems strange to talk about "use a component" when talking about the lower-level foundations. Presumably people are using sass mixins because our components don't work for them (either because it lacks functionality or they're not using React)?